docs/concepts/submit-data.md

```diff
@@ -0,0 +1,221 @@
+# Submitting Data Blobs to Celestia
+
+To submit data to celestia, users submit blob transactions (`BlobTx`). Blob
+transactions contain two components, a standard cosmos-sdk transaction called
+`MsgPayForBlobs` and a Blob of data.
+
+## Fee Market and Mempool
+
+Celestia makes use of a standard gas priced prioritized mempool. By default,
+transactions with fees higher than that of other transactions in the mempool
+will be prioritized by validators.
+
+### Fees and Gas Limits
+
+As of version v1.0.0 of the application (celestia-app), there is no enforced
+protocol minimum fee (similar to EIP1559 in Ethereum). Instead, each consensus
+node running a mempool uses a locally configured gas price threshold that must
+be breached in order for that node to accept a transaction, either from a user
+or from another consensus node, into its mempool. Members of the community have
+already expressed interest in pursuing or accepting some flavor of a protocol
+enforced minimum fee.
+
+As of version v1.0.0 of the application (celestia-app), gas is not refunded.
+Instead, transaction fees are deducted by a flat fee, originally determined by
+the user via multiplying the gas limit by the desired gas price. This means that
+users should use an accurate gas limit value if they do not wish to over pay.
+Again, community members have already expressed pursuing and accepting a
+solution to this problem.
+
+#### Estimating PFB Gas
+
+Generally, the gas used by a PFB transaction involves a static "fixed cost" and
+a dynamic cost based on the size of each blob involved in the transaction.
+
+> Note: For a general use case of a normal account submitting a PFB, the static
+> costs can be treated as such. However, due to the description above of how gas
+> works in the cosmos-sdk this is not always the case. Notably, if we use a
+> vesting account or the `feegrant` modules, then these static costs change.
+
+The "fixed cost" is an approximation of the gas consumed by operations outside
+the function `GasToConsume` (for example, signature verification, tx size, read
+access to accounts), which has a default value of 65,000.
```

Comment:
```suggestion
access to accounts), which has a default value of 65,000 utia.
```

I believe so?

Reply:
this is actually gas, which I originally left blank thinking it was mentioned in the beginning of the sentence, but I'll add it at the end too

Reply:
yeah correct